code/mob/living/critter/flockcritter_parent.dm

```diff
@@ -19,24 +19,24 @@
 	// HEALTHS
 	var/health_brute = 1
 	var/health_burn = 1
-	// no medicine for radio birds
```

Comment:
useful comment

Reply:
Not sure if its needed when its a parent variable? Does a whole lot more than no healing too

Reply:
Maybe change the comment to reflect that then. I just don't want future people wondering why we disabled metabolism without explanation